docker/gaffer-jhub-options-server/.env

```diff
@@ -0,0 +1 @@
+JHUB_OPTIONS_SERVER_VERSION=1.0.0
```

Comment:
How is this version going to be managed? Might be good to have some kind of script to increment the version for every release.

Reply:
Yeah, good question. It will need to match whatever version is in [`package.json`](https://github.com/gchq/gaffer-docker/pull/134/files#diff-25521e29fe753f154c092f887a4b6ef1bc74e80d480ff2ef7792d7178772c09eR3), so maybe I should remove the `.env` file and add something to all the CI scripts to grab the version that way?